OpenRA.Mods.Common/Traits/Explodes.cs

```diff
@@ -81,5 +90,14 @@ WeaponInfo ChooseWeaponForExplosion(Actor self)
 			var useFullExplosion = self.World.SharedRandom.Next(100) <= info.LoadedChance;
 			return (shouldExplode && useFullExplosion) ? info.WeaponInfo : info.EmptyWeaponInfo;
 		}
+
+		public void Damaged(Actor self, AttackInfo e)
+		{
+			if (info.DamageThreshold == 0)
+				return;
+
+			if ((health.HP * 100) < (info.DamageThreshold * health.MaxHP))
```

Comment:
Unnecessary parens.  You can simplify this to `if (health.HP * 100 < info.DamageThreshold * health.MaxHP)`